tests/integration/test_search_integration.py

```diff
@@ -71,8 +71,8 @@
 #         time.sleep(12)
 #         with cls.app.app_context():
 #             while True:
-#                 if index.cluster_available():
-#                     index.create_index()
+#                 if index.SearchSession.cluster_available():
```

Comment:
Should this file just get removed?

Reply:
Yeah, it's just getting more and more out of date anyway. Fixed in 969b090